ecosystem/sep-0008.md

```diff
@@ -298,13 +298,15 @@ There are two possible values for `action_method`.
    -----|------|------------
    `result` | string | `follow_next_url`.
    `next_url` | string | A URL where the user can complete the required actions with all the parameters included in the original POST pre-filled or already accepted.
+   `message` | string | (optional) A human readable string containing information regarding the further action required.
 
    Response Example:
 
    ```json
    {
      "result": "follow_next_url",
-     "next_url": "https://..."
+     "next_url": "https://...",
+     "message": "Please sign the terms of service via the provided URL."
```

Comment:
❗ Same as my other comment regarding audience:
```suggestion
     "message": "Please sign the terms of service."
```

Reply:
Addressed in the same commit: https://github.com/stellar/stellar-protocol/pull/859/commits/d251e4a86f38d7301e31e55358ae2af6e23b4815